G2P.pm

```diff
@@ -726,7 +753,7 @@ sub gene_overlap_filtering {
           foreach my $ar (@{$gene_data->{'allelic requirement'}}) {
             $self->{ar}->{$gene_stable_id}->{$ar} = 1;
           } 
-          $self->write_report('G2P_gene_data', $gene_stable_id, $gene_data, $gene_data->{'gene_xrefs'});
+          $self->write_report('G2P_gene_data', $gene_stable_id, $gene_data, $gene_data->{'gene_xrefs'},  $gene_data->{'confidence_value'});
```

Comment:
The method `write_report` is expecting the HGNC after gene_xrefs but here at that position you have the confidence_value. It should be:
`$self->write_report('G2P_gene_data', $gene_stable_id, $gene_data, $gene_data->{'gene_xrefs'},  undef, $gene_data->{'confidence_value'});`